K8SSAND-1801 ⁃ Fix flaky test

Sometimes the NonDestructiveOpsIT test fails:
```Starting testGetTables[0: 3_11](com.datastax.mgmtapi.NonDestructiveOpsIT)...
INFO  [nioEventLoopGroup-2-2] 2022-09-23 18:27:05,522 Cli.java:617 - address=/172.17.0.1:34080 url=/api/v0/probes/liveness status=200 OK
INFO  [nioEventLoopGroup-2-2] 2022-09-23 18:27:05,578 Cli.java:617 - address=/172.17.0.1:34080 url=/api/v0/lifecycle/start status=202 Accepted
INFO  [nioEventLoopGroup-2-2] 2022-09-23 18:27:05,588 Cli.java:617 - address=/172.17.0.1:34080 url=/api/v0/probes/readiness status=200 OK
INFO  [main] 2022-09-23 18:27:05,589 NonDestructiveOpsIT.java:101 - CASSANDRA ALIVE: true
INFO  [nioEventLoopGroup-2-1] 2022-09-23 18:27:10,596 Cli.java:617 - address=/172.17.0.1:44152 url=/api/v0/ops/tables status=400 Bad Request
INFO  [nioEventLoopGroup-2-1] 2022-09-23 18:27:10,603 Cli.java:617 - address=/172.17.0.1:44152 url=/api/v0/ops/tables status=200 OK
INFO  [nioEventLoopGroup-2-1] 2022-09-23 18:27:10,609 Cli.java:617 - address=/172.17.0.1:44152 url=/api/v0/ops/tables status=200 OK
FAILURE: testGetTables[0: 3_11](com.datastax.mgmtapi.NonDestructiveOpsIT)
java.util.concurrent.CompletionException: io.netty.util.IllegalReferenceCountException: refCnt: 0
	at java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:273)
	at java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:280)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:618)
	at java.util.concurrent.CompletableFuture.uniApplyStage(CompletableFuture.java:628)
	at java.util.concurrent.CompletableFuture.thenApply(CompletableFuture.java:1996)
	at com.datastax.mgmtapi.NonDestructiveOpsIT.testGetTables(NonDestructiveOpsIT.java:714)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:364)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:272)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:237)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:158)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
Caused by: io.netty.util.IllegalReferenceCountException: refCnt: 0
	at io.netty.buffer.AbstractByteBuf.ensureAccessible(AbstractByteBuf.java:1489)
	at io.netty.buffer.AbstractByteBuf.checkIndex(AbstractByteBuf.java:1418)
	at io.netty.buffer.AbstractByteBuf.checkDstIndex(AbstractByteBuf.java:1444)
	at io.netty.buffer.CompositeByteBuf.getBytes(CompositeByteBuf.java:1048)
	at io.netty.buffer.CompositeByteBuf.getBytes(CompositeByteBuf.java:49)
	at io.netty.buffer.ByteBufUtil.decodeString(ByteBufUtil.java:834)
	at io.netty.buffer.AbstractByteBuf.toString(AbstractByteBuf.java:1247)
	at io.netty.buffer.AbstractByteBuf.toString(AbstractByteBuf.java:1242)
	at com.datastax.mgmtapi.NonDestructiveOpsIT.responseAsCodeAndBody(NonDestructiveOpsIT.java:804)
	at java.util.concurrent.CompletableFuture.uniApply(CompletableFuture.java:616)
	... 45 more
=====> Showing last 100 entries of system.log
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,453 CompactionManager.java:295 - Starting Upgrade sstables for system.batchlog
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,454 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.batchlog
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,454 CompactionManager.java:295 - Starting Upgrade sstables for system.hints
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,454 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.hints
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,454 CompactionManager.java:295 - Starting Upgrade sstables for system.schema_usertypes
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,454 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.schema_usertypes
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,454 CompactionManager.java:295 - Starting Upgrade sstables for system.available_ranges
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,454 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.available_ranges
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,455 CompactionManager.java:295 - Starting Upgrade sstables for system.peer_events
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,455 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.peer_events
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,455 CompactionManager.java:295 - Starting Upgrade sstables for system.paxos
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,455 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.paxos
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,455 CompactionManager.java:295 - Starting Upgrade sstables for system.schema_columns
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,455 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.schema_columns
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,455 CompactionManager.java:295 - Starting Upgrade sstables for system.sstable_activity
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,456 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.sstable_activity
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,456 CompactionManager.java:295 - Starting Upgrade sstables for system.schema_functions
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,456 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.schema_functions
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,456 CompactionManager.java:295 - Starting Upgrade sstables for system.range_xfers
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,456 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.range_xfers
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,456 CompactionManager.java:295 - Starting Upgrade sstables for system.schema_columnfamilies
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,456 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.schema_columnfamilies
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,456 CompactionManager.java:295 - Starting Upgrade sstables for system.views_builds_in_progress
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,457 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.views_builds_in_progress
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,457 CompactionManager.java:295 - Starting Upgrade sstables for system.schema_aggregates
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,457 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.schema_aggregates
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,457 CompactionManager.java:295 - Starting Upgrade sstables for system.IndexInfo
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,457 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.IndexInfo
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,457 CompactionManager.java:295 - Starting Upgrade sstables for system.transferred_ranges
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,457 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.transferred_ranges
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,457 CompactionManager.java:295 - Starting Upgrade sstables for system.peers
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,457 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.peers
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,458 CompactionManager.java:295 - Starting Upgrade sstables for system.compaction_history
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,458 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.compaction_history
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,458 CompactionManager.java:295 - Starting Upgrade sstables for system.batches
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,458 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.batches
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,458 CompactionManager.java:295 - Starting Upgrade sstables for system.schema_keyspaces
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,458 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.schema_keyspaces
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,458 CompactionManager.java:295 - Starting Upgrade sstables for system.size_estimates
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,459 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.size_estimates
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,464 CompactionManager.java:295 - Starting Upgrade sstables for system.schema_triggers
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,464 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.schema_triggers
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,464 CompactionManager.java:295 - Starting Upgrade sstables for system.built_views
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,465 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.built_views
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,465 CompactionManager.java:295 - Starting Upgrade sstables for system.prepared_statements
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,465 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system.prepared_statements
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,465 CompactionManager.java:295 - Starting Upgrade sstables for system_schema.functions
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,465 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_schema.functions
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,465 CompactionManager.java:295 - Starting Upgrade sstables for system_schema.dropped_columns
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,466 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_schema.dropped_columns
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,466 CompactionManager.java:295 - Starting Upgrade sstables for system_schema.tables
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,466 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_schema.tables
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,466 CompactionManager.java:295 - Starting Upgrade sstables for system_schema.types
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,466 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_schema.types
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,466 CompactionManager.java:295 - Starting Upgrade sstables for system_schema.indexes
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,466 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_schema.indexes
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,467 CompactionManager.java:295 - Starting Upgrade sstables for system_schema.triggers
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,467 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_schema.triggers
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,467 CompactionManager.java:295 - Starting Upgrade sstables for system_schema.keyspaces
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,467 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_schema.keyspaces
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,467 CompactionManager.java:295 - Starting Upgrade sstables for system_schema.columns
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,467 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_schema.columns
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,468 CompactionManager.java:295 - Starting Upgrade sstables for system_schema.views
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,468 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_schema.views
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,468 CompactionManager.java:295 - Starting Upgrade sstables for system_schema.aggregates
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,468 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_schema.aggregates
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,468 CompactionManager.java:295 - Starting Upgrade sstables for system_auth.resource_role_permissons_index
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,468 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_auth.resource_role_permissons_index
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,468 CompactionManager.java:295 - Starting Upgrade sstables for system_auth.roles
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,469 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_auth.roles
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,469 CompactionManager.java:295 - Starting Upgrade sstables for system_auth.role_members
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,469 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_auth.role_members
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,469 CompactionManager.java:295 - Starting Upgrade sstables for system_auth.role_permissions
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,469 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_auth.role_permissions
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,469 CompactionManager.java:295 - Starting Upgrade sstables for system_traces.sessions
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,469 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_traces.sessions
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,469 CompactionManager.java:295 - Starting Upgrade sstables for system_traces.events
INFO  [pool-7-thread-1] 2022-09-23 18:23:08,470 CompactionManager.java:306 - No sstables to UPGRADE_SSTABLES for system_traces.events
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:23:23,595 MigrationManager.java:205 - Create new Keyspace: KeyspaceMetadata{name=getkeyspacestest, params=KeyspaceParams{durable_writes=true, replication=ReplicationParams{class=org.apache.cassandra.locator.NetworkTopologyStrategy, datacenter1=1}}, tables=[], views=[], functions=[], types=[]}
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:23:53,871 MigrationManager.java:205 - Create new Keyspace: KeyspaceMetadata{name=someTestKeyspace, params=KeyspaceParams{durable_writes=true, replication=ReplicationParams{class=org.apache.cassandra.locator.NetworkTopologyStrategy, datacenter1=1}}, tables=[], views=[], functions=[], types=[]}
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:24:09,013 MigrationManager.java:205 - Create new Keyspace: KeyspaceMetadata{name=alteringKeyspaceTest, params=KeyspaceParams{durable_writes=true, replication=ReplicationParams{class=org.apache.cassandra.locator.NetworkTopologyStrategy, datacenter1=1}}, tables=[], views=[], functions=[], types=[]}
WARN  [epollEventLoopGroup-4-3] 2022-09-23 18:24:09,042 NetworkTopologyStrategy.java:256 - Your replication factor 3 for keyspace alteringKeyspaceTest is higher than the number of nodes 1 for datacenter datacenter1
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:24:09,043 MigrationManager.java:286 - Update Keyspace 'alteringKeyspaceTest' From KeyspaceMetadata{name=alteringKeyspaceTest, params=KeyspaceParams{durable_writes=true, replication=ReplicationParams{class=org.apache.cassandra.locator.NetworkTopologyStrategy, datacenter1=1}}, tables=[], views=[], functions=[], types=[]} To KeyspaceMetadata{name=alteringKeyspaceTest, params=KeyspaceParams{durable_writes=true, replication=ReplicationParams{class=org.apache.cassandra.locator.NetworkTopologyStrategy, datacenter1=3}}, tables=[], views=[], functions=[], types=[]}
INFO  [pool-7-thread-1] 2022-09-23 18:24:54,461 CompactionManager.java:295 - Starting Cleanup for system_traces.events
INFO  [pool-7-thread-1] 2022-09-23 18:24:54,462 CompactionManager.java:491 - Skipping cleanup for 0/0 sstables for system_traces.events since they are fully contained in owned ranges ([(-5608215076967485064,-5527698873834606485], (-2977371063488082977,-2928060481510633917], (8152392837159562131,8326616592283694415], (-8547880687349964152,-8267384822208948353], (1695021220676445769,1714213293566092192], (-7435351946265648279,-7343476100702619151], (-487283330764436908,-391128750098823160], (-7141978681568792095,-7137921697564658161], (875022986836748963,1091677036486413638], (1091677036486413638,1094955618020076396], (8978716905245965504,8988756191652696740], (1117588493242794059,1454588967342780060], (-1556654912178874322,-1386506392121915095], (3283463783555941980,3393857658042264591], (2165132950418471472,2413064854035995689], (-8087883642606142626,-8076490200727224272], (-2811216078825679648,-2751494666465122273], (-8627227394766093658,-8583561323093337759], (-2751494666465122273,-2743106879165911346], (-6475864912734110790,-6392892849781338963], (-7057411207175895622,-7035356937400246404], (-7553664452124801742,-7528531716214588379], (-3024514780886363740,-3011049812801169469], (-8111996584819836778,-8087883642606142626], (5700190223821509287,5780891789292469394], (3207964296574221872,3258329276845687399], (-3160454195032285152,-3099914787241424409], (-8763452869810475044,-8742764877578511218], (-5650813696733987804,-5621302678051296322], (-3681284622888301899,-3315288412999872311], (2413064854035995689,2429967996829740985], (-553835757946510609,-551673560644937151], (120917389243609425,145616248650074401], (-6974494539939719157,-6971162302492915995], (-963024398302780996,-947856680807785722], (3887150608470734454,3995110472332082298], (6417588650986243825,6503055488571476294], (5819425025051105012,6233949790808383888], (2573182915737305807,2633872376937466173], (2513001448670148162,2528800235687867636], (5025640658552958046,5041723546979277217], (-8267384822208948353,-8196258489293215436], (3067379259240920061,3104994991404764002], (4056717163703384213,4295159225150268779], (-2853976677710555592,-2811216078825679648], (1714213293566092192,1809632812244348101], (1111299303983091769,1117588493242794059], (7464641782727846144,7478254293321465370], (-3777936620133291434,-3700791261384919827], (7603698966524409136,7621771401641359875], (-6867380236064562900,-6771186499316418237], (-2381150565164951642,-2298346630389684156], (-8196258489293215436,-8174277543715279733], (-4799357709032948037,-4679564509426326334], (5218994656627453659,5380487803940333383], (-5717192125887494164,-5650813696733987804], (-5227478939295583926,-5044076271095223514], (7052682994720437280,7077038240592216195], (7077038240592216195,7154032139743027133], (-3011049812801169469,-2977371063488082977], (2826403761532095337,2897116872557874615], (5041723546979277217,5218994656627453659], (2528800235687867636,2566643872963007162], (-9208435200405316342,-9107950504055187658], (-2611650794602031271,-2594095441583729418], (-6722523149215413733,-6663042022792613414], (-947856680807785722,-922341921851675634], (-2384568389989215616,-2381150565164951642], (-8119318880554564454,-8115217757872979790], (-8159966036869443884,-8119318880554564454], (8974079340447074114,8978716905245965504], (-1922656235551121543,-1863166383544664093], (6233949790808383888,6402832814429491875], (-4064635164372722718,-4022251378311204354], (7795642702254803514,7872302897020860568], (789472890556321126,875022986836748963], (9121703454156045534,9128366342714339347], (-2594095441583729418,-2472644865746073538], (1937641191420982944,1996697025613626067], (-3315288412999872311,-3222709072408553504], (2566643872963007162,2573182915737305807], (3000932893276091911,3029762363526105966], (3998302284997155725,4056717163703384213], (4358683984536410347,4394779092118259609], (-1674094895800948301,-1597407607286697804], (-3099914787241424409,-3024514780886363740], (5546937084546243068,5588225556828743328], (7478254293321465370,7502140025462409648], (-114233345188358437,-113034579893932351], (3029762363526105966,3067379259240920061], (-7927425418072237698,-7706752773783329601], (715738750676425748,789472890556321126], (2703088464253369374,2779482115937670491], (2429967996829740985,2433315431567590686], (8133474614031802704,8152392837159562131], (2633872376937466173,2645737376202216648], (-1863166383544664093,-1754562895315506517], (8959232717745355937,8974079340447074114], (3995110472332082298,3998302284997155725], (-8561733512044986313,-8547880687349964152], (-551673560644937151,-487283330764436908], (512663282056833450,673729549840242919], (-6020650256058426559,-5939788780330314285], (7268447198123163585,7293908664980259947], (9128366342714339347,9170325517484999781], (-8742764877578511218,-8627227394766093658], (-7298179987992969659,-7163037268517181557], (4877337096193634537,4986709461993600933], (7178783722934114692,7191086799100022548], (-1597407607286697804,-1576857307840001692], (-8076490200727224272,-7927425418072237698], (8109085601174622409,8110026519244871520], (3790857157922185486,3887150608470734454], (145616248650074401,162352104939955848], (2995420486442947665,3000932893276091911], (-5375916151421949573,-5331439940178997397], (-1745401912259099184,-1674094895800948301], (-2644078766885129061,-2622416299505241193], (-6771186499316418237,-6722523149215413733], (-5331439940178997397,-5227478939295583926], (4710974724637535098,4738187956331570675], (8326616592283694415,8380012034088702852], (-5527698873834606485,-5375916151421949573], (-139698329559064046,-114233345188358437], (-6663042022792613414,-6534346064594747827], (2945902609934112118,2995420486442947665], (5588225556828743328,5700190223821509287], (-2622416299505241193,-2611650794602031271], (-2424323378663846255,-2399350911698801705], (-8902668047853959924,-8763452869810475044], (-4150495571126015273,-4064635164372722718], (4986709461993600933,5025640658552958046], (-8115217757872979790,-8111996584819836778], (-7522080339074361384,-7505299979186473090], (-5044076271095223514,-4799357709032948037], (7171280523463936541,7178783722934114692], (2481877356003268015,2513001448670148162], (673729549840242919,715738750676425748], (-7706752773783329601,-7575774752804557303], (-7343476100702619151,-7328870080866127680], (8083083142728414921,8109085601174622409], (162352104939955848,196741643106222894], (-113034579893932351,95008770031596217], (2433315431567590686,2481877356003268015], (-4361558228404318119,-4150495571126015273], (-3842154230750862839,-3784082589135579655], (-1972837995952449325,-1922656235551121543], (8569592337924163210,8598568432877462747], (-6966804498026076459,-6953014987148171335], (-662326554807482294,-656905488241500068], (4394779092118259609,4533703730566692416], (-5939788780330314285,-5884345637665192182], (9170325517484999781,-9208435200405316342], (-2442735201430424731,-2424323378663846255], (-4575690913517294632,-4405840219563383914], (502310920411852829,512663282056833450], (-391128750098823160,-139698329559064046], (8002151580237924095,8070716396379914151], (-1016787557745676889,-963024398302780996], (4738187956331570675,4877337096193634537], (6705229200380692198,6706016556454247531], (7191086799100022548,7238520072464504807], (7985998031605999517,8002151580237924095], (8784028818034363280,8917172335298836596], (242044634748524532,480382760354554888], (-5884345637665192182,-5717192125887494164], (-9107950504055187658,-8976876678322159794], (2779482115937670491,2826403761532095337], (7238520072464504807,7268447198123163585], (7621771401641359875,7795642702254803514], (7872302897020860568,7878568500793345613], (8598568432877462747,8665041109451835835], (-6100145699143020448,-6020650256058426559], (3104994991404764002,3207964296574221872], (-7505299979186473090,-7470506453148885170], (-1386506392121915095,-1120572414886587045], (-4679564509426326334,-4596701534616370901], (-691734634846286320,-672833944397074581], (7502140025462409648,7555303963614892362], (8437194924832415244,8569592337924163210], (-7528531716214588379,-7522080339074361384], (-2928060481510633917,-2853976677710555592], (-656905488241500068,-644009253005451014], (-3700791261384919827,-3681284622888301899], (-6392892849781338963,-6320741545463452351], (-2246090409892780393,-1972837995952449325], (-644009253005451014,-553835757946510609], (6706016556454247531,7052682994720437280], (3258329276845687399,3283463783555941980], (7293908664980259947,7299759714732949686], (-6134271020779602631,-6100145699143020448], (4302374287247878441,4358683984536410347], (2645737376202216648,2703088464253369374], (-1754562895315506517,-1745401912259099184], (2897116872557874615,2945902609934112118], (-6971162302492915995,-6966804498026076459], (-2281462001518277156,-2246090409892780393], (1094955618020076396,1111299303983091769], (5380487803940333383,5509074274512287097], (5780891789292469394,5819425025051105012], (-6320741545463452351,-6134271020779602631], (6503055488571476294,6550242341273498981], (-8976876678322159794,-8902668047853959924], (8070716396379914151,8083083142728414921], (8666749346464921789,8694899364579613165], (-4022251378311204354,-3842154230750862839], (-7575774752804557303,-7553664452124801742], (7878568500793345613,7985998031605999517], (-4405840219563383914,-4361558228404318119], (-7163037268517181557,-7141978681568792095], (-4596701534616370901,-4575690913517294632], (8110026519244871520,8133474614031802704], (-3784082589135579655,-3777936620133291434], (-7470506453148885170,-7435351946265648279], (8380012034088702852,8437194924832415244], (-1576857307840001692,-1556654912178874322], (-7137921697564658161,-7057411207175895622], (-5621302678051296322,-5608215076967485064], (8694899364579613165,8784028818034363280], (-6534346064594747827,-6475864912734110790], (7555303963614892362,7603698966524409136], (-7310119524892340928,-7298179987992969659], (-2298346630389684156,-2281462001518277156], (8917172335298836596,8959232717745355937], (-2657477469671500923,-2644078766885129061], (-8583561323093337759,-8561733512044986313], (-3222709072408553504,-3160454195032285152], (5509074274512287097,5546937084546243068], (7299759714732949686,7464641782727846144], (-1120572414886587045,-1016787557745676889], (-8174277543715279733,-8159966036869443884], (480382760354554888,502310920411852829], (-7035356937400246404,-6974494539939719157], (6550242341273498981,6705229200380692198], (-6934655201110496413,-6867380236064562900], (6402832814429491875,6417588650986243825], (3393857658042264591,3790857157922185486], (-2472644865746073538,-2442735201430424731], (225891950630660354,242044634748524532], (-6953014987148171335,-6934655201110496413], (-866409253657853135,-691734634846286320], (-7328870080866127680,-7310119524892340928], (-2743106879165911346,-2657477469671500923], (-922341921851675634,-866409253657853135], (8988756191652696740,9121703454156045534], (1454588967342780060,1695021220676445769], (-2399350911698801705,-2384568389989215616], (196741643106222894,225891950630660354], (4295159225150268779,4302374287247878441], (7154032139743027133,7171280523463936541], (8665041109451835835,8666749346464921789], (-672833944397074581,-662326554807482294], (1809632812244348101,1937641191420982944], (4533703730566692416,4710974724637535098], (1996697025613626067,2165132950418471472], (95008770031596217,120917389243609425]])
INFO  [pool-7-thread-1] 2022-09-23 18:24:54,462 CompactionManager.java:306 - No sstables to CLEANUP for system_traces.events
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:25:24,681 ColumnFamilyStore.java:744 - Loading new SSTables for system_traces/events...
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:25:24,682 ColumnFamilyStore.java:823 - No new SSTables were found for system_traces/events
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:25:39,789 CompactionManager.java:295 - Starting Remove deleted data for system_traces.events
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:25:39,791 CompactionManager.java:306 - No sstables to GARBAGE_COLLECT for system_traces.events
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:26:25,186 MigrationManager.java:205 - Create new Keyspace: KeyspaceMetadata{name=getreplicationtest, params=KeyspaceParams{durable_writes=true, replication=ReplicationParams{class=org.apache.cassandra.locator.NetworkTopologyStrategy, datacenter1=1}}, tables=[], views=[], functions=[], types=[]}
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:26:55,475 LogbackLoggingSupport.java:117 - set log level to DEBUG for classes under 'org.apache.cassandra.cql3' (if the level doesn't look like 'debug' then the logger couldn't parse 'debug')
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:26:55,477 LogbackLoggingSupport.java:117 - set log level to DEBUG for classes under 'org.apache.cassandra.auth' (if the level doesn't look like 'debug' then the logger couldn't parse 'debug')
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:26:55,478 LogbackLoggingSupport.java:117 - set log level to DEBUG for classes under 'org.apache.cassandra.batchlog' (if the level doesn't look like 'debug' then the logger couldn't parse 'debug')
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:26:55,479 LogbackLoggingSupport.java:117 - set log level to DEBUG for classes under 'org.apache.cassandra.net.ResponseVerbHandler' (if the level doesn't look like 'debug' then the logger couldn't parse 'debug')
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:26:55,480 LogbackLoggingSupport.java:117 - set log level to DEBUG for classes under 'org.apache.cassandra.service.AbstractReadExecutor' (if the level doesn't look like 'debug' then the logger couldn't parse 'debug')
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:26:55,482 LogbackLoggingSupport.java:117 - set log level to DEBUG for classes under 'org.apache.cassandra.service.AbstractWriteResponseHandler' (if the level doesn't look like 'debug' then the logger couldn't parse 'debug')
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:26:55,483 LogbackLoggingSupport.java:117 - set log level to DEBUG for classes under 'org.apache.cassandra.service.paxos' (if the level doesn't look like 'debug' then the logger couldn't parse 'debug')
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:26:55,484 LogbackLoggingSupport.java:117 - set log level to DEBUG for classes under 'org.apache.cassandra.service.ReadCallback' (if the level doesn't look like 'debug' then the logger couldn't parse 'debug')
INFO  [epollEventLoopGroup-4-3] 2022-09-23 18:26:55,484 LogbackLoggingSupport.java:117 - set log level to DEBUG for classes under 'org.apache.cassandra.service.ResponseResolver' (if the level doesn't look like 'debug' then the logger couldn't parse 'debug')
=====> End of last 100 entries of system.log```



┆Issue is synchronized with this [Jira Task](https://k8ssandra.atlassian.net/browse/K8SSAND-1801) by [Unito](https://www.unito.io)
┆friendlyId: K8SSAND-1801
┆priority: Medium

--- a/management-api-server/src/test/java/com/datastax/mgmtapi/NonDestructiveOpsIT.java
+++ b/management-api-server/src/test/java/com/datastax/mgmtapi/NonDestructiveOpsIT.java
@@ -931,7 +931,7 @@ public class NonDestructiveOpsIT extends BaseDockerIntegrationTest {
     return null;
   }
 
-  private Pair<Integer, String> responseAsCodeAndBody(FullHttpResponse r) {
-    return Pair.of(r.status().code(), r.content().retain().toString(UTF_8));
+  private synchronized Pair<Integer, String> responseAsCodeAndBody(FullHttpResponse r) {
+    return Pair.of(r.status().code(), r.content().toString(UTF_8));
   }
 }

--- a/management-api-server/src/test/java/com/datastax/mgmtapi/NonDestructiveOpsIT.java
+++ b/management-api-server/src/test/java/com/datastax/mgmtapi/NonDestructiveOpsIT.java
@@ -932,6 +932,6 @@ public class NonDestructiveOpsIT extends BaseDockerIntegrationTest {
   }
 
   private Pair<Integer, String> responseAsCodeAndBody(FullHttpResponse r) {
-    return Pair.of(r.status().code(), r.content().toString(UTF_8));
+    return Pair.of(r.status().code(), r.content().retain().toString(UTF_8));
   }
 }

